apps/condo/domains/acquiring/components/payments/PaymentsTable.tsx

```diff
@@ -92,8 +106,11 @@ const PaymentsTable: React.FC<IPaymentsTableProps> = ({ billingContext, contexts
     const [dateRange, setDateRange] = useDateRangeSearch('advancedAt', loading)
 
     useEffect(() => {
-        if (!dateRange && appliedFiltersCount < 1 && !search) {
+        if (!hasFilters && shouldApplyDefaultFilter && !isDefaultFilterApplied) {
+            isDefaultFilterApplied = true
```

Comment:
this would affect all instances of this interface, are you sure you want such behaviour ?

Reply:
This hook works only once (`deps = []`) within the table component. It's like old-style `componentDidMount` hook.